skrifa/src/outline/mod.rs

```diff
@@ -185,6 +189,11 @@ impl<'a> DrawSettings<'a> {
         self.memory = memory;
         self
     }
+
+    pub fn with_offcurve_first_mode(mut self, off_curve_first_mode: ToPathStyle) -> Self {
```

Comment:
Not super happy with the naming and I feel like this also deserves a doc comment. Alternatively, make it `#[doc(hidden)]` so we don't even confuse users who have no idea why they'd want this

Reply:
Oops, let me rename it to match ToPathStyle. I don't think it should be hidden. Will add a comment.

Reply:
Fixed both issues.